### ISSUE TITLE
AI workflow help with tagging?

### ISSUE THREAD

**Issue by danielbayley** [lucide-icons/lucide#1345]
I feel like tagging is something some kind of AI based workflow could really help with… (Iv’e otherwise found ChatGTP etc. to be of limited use so far, given how frequently wrong it is…) But not sure what that would like exactly… Any ideas?

**Comment by karsa-mistmere** [comment]
I'm mostly using a thesaurus (actually multiple thesauri) and sometimes Google reverse image search to get inspired by synonyms, antonyms as well as related terms/subjects. Not sure AI is "there" yet to help us with this.

Although I've given ChatGPT a try and it's not half bad: 
![image](https://github.com/lucide-icons/lucide/assets/17746067/900b5e02-6138-44a7-aa84-1116b0b2995e)

Maybe we can get a sponsorship. 😆

**Comment by danielbayley** [comment]
> I'm mostly using a thesaurus (actually multiple **thesauri**)

> thesauri

@karsa-mistmere Clearly… 🤓😂

Iv’e just been using my meat brain… old school.

> Although I've given ChatGPT a try and it's not half bad:

> Maybe we can get a sponsorship. 😆

Yeah, I was thinking along the lines of some kind of language model to offer up suggestions based on the file name, as part of the edit page (proposed in #1022) @jguddas

**Comment by ericfennis** [comment]
@danielbayley Really interesting idea, I'm open to exploring this more. The list from @karsa-mistmere is quite impressive. I'm thinking of getting a license for GPT-4, so maybe that can be even better. About creating a custom model for this is possible but sounds like a lot of work to create, I think it's easier to stick with something that already exists like: GPT. But I'm not an Machine Learning specialist

**Comment by danielbayley** [comment]
> The list from @karsa-mistmere is quite impressive. I'm thinking of getting a license for GPT-4, so maybe that can be even better

> About creating a custom model for this is possible but sounds like a lot of work to create, I think it's easier to stick with something that already exists like: GPT.

@ericfennis Yeah for something as simple as this, basically just word association—something a _language model_ should excel at—I think that would be overkill, and pretty much any will do the job here… GPT-4 might not even be necessary. Maybe something open source could even work for this?

**Comment by ericfennis** [comment]
Closed by #3372
